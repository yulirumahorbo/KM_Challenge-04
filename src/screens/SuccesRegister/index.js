import {
  StyleSheet, Text, View, SafeAreaView, Image,
} from 'react-native';
import React from 'react';
import { moderateScale } from 'react-native-size-matters';
import { TouchableOpacity } from 'react-native-gesture-handler';
import CheckIcon from '../../assets/image/Check_Icon.png';
import iconBack from '../../assets/image/icon-back.png';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#ccaf9b',
  },
  title: {
    marginBottom: moderateScale(8),
    fontSize: moderateScale(20),
    color: 'white',
    fontWeight: '700',
  },
  regularText: {
    fontWeight: '500',
    fontSize: moderateScale(18),
    color: '#000000',
  },
  regularSubText: {
    fontWeight: '400',
    fontSize: moderateScale(16),
    color: '#000000',
  },
  buttonContainer: {
    backgroundColor: '#873c1e',
    paddingHorizontal: moderateScale(115),
    paddingVertical: moderateScale(10),
    borderRadius: moderateScale(30),
  },
});
function SuccesRegister({ navigation }) {
  return (
    <SafeAreaView style={styles.container}>
      <View
        style={{
          flexDirection: 'row',
          alignItems: 'center',
          margin: moderateScale(16),
        }}
      >
        <TouchableOpacity onPress={() => navigation.navigate('Login')}>
          <Image
            source={iconBack}
            style={{ height: moderateScale(35), width: moderateScale(35) }}
          />
        </TouchableOpacity>
        <Text style={styles.regularText}>Login</Text>
      </View>
      <View
        style={{
          backgroundColor: '#ccaf9b',
          alignItems: 'center',
          paddingTop: moderateScale(80),
        }}
      >
        <Text style={[styles.title, { color: '#873c1e' }]}>
          Registration Completed!
        </Text>
        <Image
          source={CheckIcon}
          style={{
            height: moderateScale(130),
            width: moderateScale(150),
            resizeMode: 'contain',
            marginTop: moderateScale(80),
          }}
        />
        <Text
          style={[
            styles.regularText,
            { width: moderateScale(250), textAlign: 'center' },
          ]}
        >
          We sent email verification to your email
        </Text>
      </View>
    </SafeAreaView>
  );
}

export default SuccesRegister;
