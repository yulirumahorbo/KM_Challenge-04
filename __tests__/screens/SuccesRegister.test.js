import React from 'react';
import { create } from 'react-test-renderer';
import ContainerTesting from '../../src/helpers/reduxTesting';
import SuccesRegister from '../../src/screens/SuccesRegister';

describe('Snapshot', () => {
  test('SuccesRegister Snapshot', async () => {
    const component = create(ContainerTesting(<SuccesRegister />));
    expect(component).toMatchSnapshot();
  });
});
