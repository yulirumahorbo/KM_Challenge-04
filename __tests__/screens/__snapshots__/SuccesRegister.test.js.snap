// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot SuccesRegister Snapshot 1`] = `null`;
